expenser-ui/src/app/create-dialog/income-form.tsx

```
@@ -0,0 +1,320 @@
+        form.reset(initialData);
+        if (initialData.category) {
+          const categoryOption = filtered.find(
+            (category) => category.name === initialData.category
+          );
+          if (categoryOption) {
+            form.setValue("category", categoryOption.id);
+          }
+
+          if (initialData.amount) {
+            const numericValue = parseFloat(initialData.amount);
+            if (!isNaN(numericValue) && numericValue > 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)